Simplenote/Classes/UIKitConstants.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+import UIKit
```

Comment:
Is this import needed?

Reply:
(Just for the record!!) Xcode actually complains if the import is missing, in the follow up PR.

Apparently SAMKeychain imported UIKit for us, and we didn't need the explicit import declarations. Error involves `CGFloat` not being in scope.

Thank you sir!!